CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 ## [[UnreleasedUniFFIVersion]] (backend crates: [[UnreleasedBackendVersion]]) - (_[[ReleaseDate]]_)
 
+### What's new?
+
+- Lift errors will not cause an abort when `panic=abort` is set.
+- Added the `cargo_metadata` feature, which is on by default.  In some cases, this can be disabled
+  for better compatibilyt with projects that don't use cargo.
```

Comment:
```suggestion
  for better compatibility with projects that don't use cargo.
```

Also maybe mention https://github.com/mozilla/uniffi-rs/pull/2182 ? Something like "In Kotlin bindings the more efficient `Enum.entries` property will be used instead of `Enum.values()` when possible". Sorry, I forgot to add this myself in the PR.